documentation/src/main/resources/pages/ditto/baic-auth-checkpermissions.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Checking Permissions for Resources  
+keywords: permissions, authorization, resources, policy, checkPermissions  
+tags: [model]  
+permalink: check-permissions.html  
```

Comment:
the permalink name should be the same as the filename
And the filename contains a typo: `baic-auth-checkpermissions.md`